### PR TITLE
Add support for Preact

### LINKED ISSUE #28: How to intergrate preactjs with this plugin
I don't know how to intergrate preactjs with this plugin. Please tell me a way to do it.

### PR DESCRIPTION
**Pull request by aulisius**
* Added `unpkg` links for `preact` and `preact-compat`
* Also closes https://github.com/mastilver/dynamic-cdn-webpack-plugin/issues/28

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -214,5 +214,23 @@
         "production": "https://unpkg.com/zone.js@[version]/dist/zone.min.js"
       }
     }
+  },
+  "preact": {
+    "var": "preact",
+    "versions": {
+      ">= 1.2.0": {
```
Comment (mastilver) [inline modules.json]: Should be `>= 2.4.0` as previous versions don't have minified bundle

File: modules.json [hunk 2]
```diff
@@ -214,5 +214,23 @@
...
+        "development": "https://unpkg.com/preact@[version]/dist/preact.js",
+        "production": "https://unpkg.com/preact@[version]/dist/preact.min.js"
+      }
+    }
+  },
+  "preact-compat": {
+    "var": "PreactCompat",
```
Comment (mastilver) [inline modules.json]: `preactCompat`

File: modules.json [hunk 3]
```diff
@@ -214,5 +214,23 @@
...
+    "versions": {
+      ">= 0.1.0": {
```
Comment (mastilver) [inline modules.json]: `>= 1.0.0`

**Review by mastilver — CHANGES_REQUESTED** [review]
Thank you for the PR, just a few changes to make the test passes

File: modules.json [hunk 4]
```diff
@@ -214,5 +214,22 @@
...
+    "var": "preact",
+    "versions": {
+      ">= 2.4.0": {
+        "development": "https://unpkg.com/preact@[version]/dist/preact.js",
+        "production": "https://unpkg.com/preact@[version]/dist/preact.min.js"
+      }
+    }
+  },
+  "preact-compat": {
+    "versions": {
```
Comment (mastilver) [inline modules.json]: `"var": "preactCompat"`

**Review by mastilver — CHANGES_REQUESTED** [review]

File: modules.json (hunk 4 above)
Comment (aulisius) [reply modules.json]: What is the use of `var` technically?  Is it the namespace under which it is exported?

**Review by mastilver — APPROVED** [review]

**Comment by mastilver** [comment]
released on `v3.1.1`

**Comment by aulisius** [comment]
Awesome. Thanks for merging 🎊

**Pull request merged**